libcloud/compute/drivers/dimensiondata.py

```diff
@@ -2015,12 +2032,19 @@ def _to_firewall_rule(self, element, locations, network_domain):
     def _to_firewall_address(self, element):
         ip = element.find(fixxpath('ip', TYPES_URN))
         port = element.find(fixxpath('port', TYPES_URN))
+        port = element.find(fixxpath('port', TYPES_URN))
+        port_list = element.find(fixxpath('portList', TYPES_URN))
+        address_list = element.find(fixxpath('ipAddressList', TYPES_URN))
         return DimensionDataFirewallAddress(
             any_ip=ip.get('address') == 'ANY',
             ip_address=ip.get('address'),
             ip_prefix_size=ip.get('prefixSize'),
             port_begin=port.get('begin') if port is not None else None,
```

Comment:
Just curious on these lines...
.get() returns None if it doesn't exist I believe, so wondering what the if is not None else None is for.